src/sasctl/pzmm/write_json_files.py

```diff
@@ -355,24 +352,20 @@ def writeBaseFitStat(
             * C = Area Under ROC
 
         Parameters
-        ---------------
+        ----------
         csvPath : string, optional
-            Location for an input CSV file that contains parameter statistics.
-            The default value is None.
+            Location for an input CSV file that contains parameter statistics. The default value is None.
         jPath : string, optional
-            Location for the output JSON file. The default value is the current
-            working directory.
+            Location for the output JSON file. The default value is the current working directory.
         userInput : boolean, optional
             If true, prompt the user for more parameters. The default value is false.
         tupleList : list of tuples, optional
-            Input parameter tuples in the form of (parameterName,
-            parameterLabel, parameterValue, dataRole). For example,
-            a sample parameter call would be ('NObs', 'Sum of Frequencies',
-            3488, 'TRAIN'). Variable dataRole is typically either TRAIN,
-            TEST, or VALIDATE or 1, 2, 3 respectively. The default value is None.
+            Input parameter tuples in the form of (parameterName, parameterLabel, parameterValue, dataRole). For
+            example, a sample parameter call would be ('NObs', 'Sum of Frequencies', 3488, 'TRAIN'). Variable dataRole
+            is typically either TRAIN, TEST, or VALIDATE or 1, 2, 3 respectively. The default value is None.
 
         Yields
-        ---------------
+        ------
```

Comment:
Same issue as with writeVarJSON() docstring, the Yields section is inappropriate here.

Reply:
Resolved.